src/err/mod.rs

```diff
@@ -265,6 +264,11 @@ impl PyErr {
     /// expected to have been set, for example from [`PyErr::occurred`] or by an error return value
     /// from a C FFI function, use [`PyErr::fetch`].
     pub fn take(py: Python<'_>) -> Option<PyErr> {
+        Self::_take(py)
+    }
+
+    #[cfg(not(Py_3_12))]
+    fn _take(py: Python<'_>) -> Option<PyErr> {
```

Comment:
Just to make sure I understand this correctly. The `_take` layer is only used to avoid repeating the doc comment of the public function for `Py_3_12` and `not(Py_3_12)`?

Reply:
Yes, I figured it was more DRY this way.